src/doc/python_scripting/functions.rst

```diff
@@ -1899,21 +1816,17 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineVectorExpression function returns 1 on success and 0 on failure.
```

Comment:
There is probably some additional detail needed here regarding 2D vs 3D vector construction.